praw/objects.py

```diff
@@ -441,7 +441,31 @@ def refresh(self):
         elif isinstance(self, Comment):
             sub = Submission.from_url(self.reddit_session, self.permalink,
                                       params={'uniq': unique})
-            other = sub.comments[0]
+            try:
+                other = sub.comments[0]
+            except IndexError:
+                # comment is "specially deleted", a reddit inconsistency;
+                # see #519, #524, #535, #537, and #552 it needs to be
+                # retreived via /api/info, but that's okay since these
+                # specially deleted comments always have the same json
+                # structure. The unique count needs to be updated
+                # in case the comment originally came from /api/info
+                msg = ("Comment {0} was deleted or removed, and had "
+                       "no replies when such happened, so a second "
+                       "request was made to /api/info.".format(self.name))
+                unique = self.reddit_session._unique_count
+                self.reddit_session._unique_count += 1
+                info = self.reddit_session.get_info(thing_id=self.name,
+                                                    params={'uniq': unique})
+                other = next(info)
+                oldkeys = set(self.__dict__.keys())
+                newkeys = set(other.__dict__.keys())
+                keydiff = ", ".join(oldkeys - newkeys)
+                if keydiff:
+                    msg += "\n" + "Could not retrieve:\n" + keydiff
```

Comment:
Can you use the format string here (e.g., `'\nCould not... : {0}'.format(keydiff)`)?